shuup/core/models/_currencies.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shuup.
+#
+# Copyright (c) 2012-2016, Shoop Commerce Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+from __future__ import unicode_literals
+
+import babel
+from django.core.exceptions import ValidationError
+from django.core.validators import MaxValueValidator, MinLengthValidator
+from django.db import models
+from django.utils.encoding import force_text, python_2_unicode_compatible
+from django.utils.translation import ugettext_lazy as _
+from parler.models import TranslatedFields
+
+from shuup.core.models import TranslatableShuupModel
+from shuup.utils.analog import define_log_model
+
+
+@python_2_unicode_compatible
+class Currency(TranslatableShuupModel):
+    identifier_attr = 'code'
+
+    code = models.CharField(verbose_name=_("code"),
+                            max_length=3, unique=True,
+                            editable=True,
+                            validators=[MinLengthValidator(3)],
+                            help_text=_("The ISO-4217 code of the currency"))
+
+    decimal_places = models.PositiveSmallIntegerField(verbose_name=_("decimal places"),
+                                                      validators=[MaxValueValidator(10)],
+                                                      default=2)
+
+    translations = TranslatedFields(
+        name=models.CharField(verbose_name=_("name"), max_length=64),
```

Comment:
@tulimaki means this `name=models.CharField(verbose_name="name", max_length=64)`